schema/device.go

```diff
@@ -12,6 +12,13 @@ const (
 	DeviceModelRubix        = "Rubix"
 )
 
+type DeviceKey struct {
+	Type  string `json:"type" default:"string"`
+	Title string `json:"title" default:"Device Key"`
+	Min   int    `json:"minLength" default:"32"`
+	Max   int    `json:"maxLength" default:"32"`
```

Comment:
@maharjanaman same thing re the min/max spelling. please replace with the correct spelling

Reply:
@Shiny380 Got it. Will make the changes. But other modules is using the same https://github.com/NubeIO/module-core-lorawan/blob/0b6ba552f1b79e89fd5d13de86aa43fd5eac28f7/schema/device.go#L43 and some struct in loraraw is still using same convention. May be something to refactor later.

Reply:
Json property, `minLength` and `maxLength` is the same. I checked with UI and it's working fine.

Reply:
ok thanks!
I just knew that `max`/`min` didn't work (which we made a few mistakes in the past) and `maximum`/`minimum` did, interesting that this also works

Reply:
if it works then no need to change!